includes/course-theme/class-sensei-course-theme-lesson.php

```diff
@@ -261,10 +261,18 @@ private function maybe_add_not_enrolled_notice() {
 				],
 			];
 
+			$notice_text  = __( 'Please register or sign in to access the course content.', 'sensei-lms' );
+			$notice_title = __( 'You don\'t have access to this lesson', 'sensei-lms' );
+
+			if ( Sensei_Utils::is_preview_lesson( $lesson_id ) ) {
+				$notice_text  = __( 'Register or sign in to take this lesson.', 'sensei-lms' );
+				$notice_title = __( 'This is a preview lesson', 'sensei-lms' );
+			}
+
 			$notices->add_notice(
 				'locked_lesson',
-				__( 'Please register or sign in to access the course content.', 'sensei-lms' ),
-				__( 'You don\'t have access to this lesson', 'sensei-lms' ),
+				$notice_text,
+				$notice_title,
 				$actions,
 				'lock'
```

Comment:
Let's also change the icon to the preview eye.

Reply:
Good idea! https://github.com/Automattic/sensei/pull/4778/commits/72f947f006db471426af8e6ea70b3edf347a1a27